x-pack/plugins/security/server/authentication/providers/kerberos.ts

```diff
@@ -75,11 +77,8 @@ export class KerberosAuthenticationProvider extends BaseAuthenticationProvider {
       return AuthenticationResult.notHandled();
     }
 
-    let authenticationResult = authorizationHeader
```

Comment:
__note:__ I'm not sure why we weren't checking the `canStartNewSession` here before, but I cannot find any reason why we shouldn't do it now.